rero_ils/modules/api.py

```diff
@@ -119,6 +120,29 @@ def get_record_by_pid(self, pid, fields=None):
             raise NotFoundError(f'Record not found pid: {pid}')
         return response.hits.hits[0]._source
 
+    def get_records_by_terms(self, terms, key='pid', fields=None,
```

Comment:
We need to discuss if we want to use python types in our project.

Reply:
ok, removed